### DISCUSSION TITLE
Example sentence for 勝ち目 and 占有 is a (confusing) direct translation of an English set phrase

### DISCUSSION THREAD

**Discussion by Morgawr** [Jitendex/Jitendex#42] [Errors and Corrections]
JP: 現実の占有は九分の勝ち目。

EN: Possession is nine-tenths of the law.

---

When I stumbled upon this entry, I read the Japanese phrase first and I had no idea what it was talking about. Then I read the English translation and I was still a confused about its meaning. I did not know the [expression itself](https://en.wikipedia.org/wiki/Possession_is_nine-tenths_of_the_law) so I was missing some of the context behind it. 

I asked a couple of Japanese friends who were also not aware of the English phrase and they were as confused by the example Japanese sentence as I was, since it takes a somewhat literal translation from English and loses all the cultural context behind it. I don't think it's a great example sentence to have unfortunately.

**Comment by stephenmk** [comment]
That sentence (174865) is currently the priority example for three terms: 占有, 九分, and 勝ち目.

I moved the priority flag for 勝ち目 to a different sentence (217384).

> こんな一流選手が相手では勝ち目がない。
> We have no chance against those top players.

The other two terms (占有 and 九分) don't have other sentences in the corpus. We could either keep or remove the priority flags. The sentence definitely looks like a direct translation of the English phrase. It doesn't seem to be used in Japanese outside discussions of the English text. If @jimmubreen agrees, I'll just remove the flags.

**Comment by JMdictProject** [comment]
I agree that it's not a great example sentence, but it's not wrong either. Until we can actually get better alternative sentences, I'd prefer the tags to stay for those two terms.
